foundation_cms/blocks/base_card_block.py

```diff
@@ -7,13 +7,9 @@
 
 
 class BaseCardBlock(BaseBlock):
-    label = blocks.CharBlock(
-        required=False, max_length=50, label="Label", help_text="Appears above the headline (max 36 characters)."
-    )
-    headline = blocks.CharBlock(
-        required=False, max_length=50, label="Headline", help_text="Appears as the main heading (max 36 characters)."
-    )
-    image = ImageChooserBlock(required=False, label="Image", help_text="Image should follow a 2:3 aspect ratio.")
+    label = blocks.CharBlock(required=False, label="Label", help_text="Appears above the headline.")
+    headline = blocks.CharBlock(required=False, label="Headline", help_text="Appears as the main heading.")
+    image = ImageChooserBlock(required=False, label="Image", help_text="Optional Image for the card")
```

Comment:
As mentioned in the description, these fields were updated after discussing with @robdivincenzo !